packages/runtime-vapor/src/directive.ts

```diff
@@ -1,6 +1,7 @@
 import { isFunction } from '@vue/shared'
 import { currentInstance, type ComponentInternalInstance } from './component'
 import { effect } from './scheduler'
+import { getExposeProxy } from '@vue/runtime-core'
```

Comment:
We should not import `runtime-core` module. ATM we should just copy them from `runtime-core`.